src/hardware/gus.cpp

```diff
@@ -955,32 +949,7 @@ void Gus::PrintStats()
 		        ratio_8bit, used_8bit_voices, ratio_16bit,
 		        used_16bit_voices);
 	}
-
-	// Calculate and print info about the volume
-	const auto scalar = std::max(mixer_level->left,
-	                             mixer_level->right);
-	const auto peak_sample = std::max(peak.left, peak.right);
-	auto peak_ratio = scalar * peak_sample / AUDIO_SAMPLE_MAX;
-
-	// It's expected and normal for multi-voice audio to periodically
-	// accumulate beyond the max, which is gracefully scaled without
-	// distortion, so there is no need to recommend that users scale-down
-	// their GUS mixer settings.
-	peak_ratio = std::min(peak_ratio, 1.0f);
-	LOG_MSG("GUS: Peak amplitude reached %.0f%% of max",
-	        100 * static_cast<double>(peak_ratio));
-
-	// Make a suggestion if the peak volume was well below 3 dB, but without
-	// the influence of the mixer's scalar because the user might have
-	// deliberately set it low.
-	if (peak_ratio / scalar < 0.6f) {
-		const auto multiplier = static_cast<uint16_t>(
-		        100 * scalar / peak_ratio);
-		LOG_MSG("GUS: If it should be louder, %s %u",
-		        fabs(scalar - 1.0f) > 0.01f ? "adjust mixer gus to"
-		                                          : "use: mixer gus",
-		        multiplier);
-	}
+	soft_limiter.PrintStats(mixer_level);
```

Comment:
Should be able to get rid of mixer level here because the limiter hold a reference to it.